libsignal-service/src/push_service.rs

```diff
@@ -512,6 +512,8 @@ pub enum ServiceError {
     NotFoundError,
 }
 
+pub(crate) const NO_ADDITIONAL_HEADERS: &[(&str, &str)] = &[];
```

Comment:
Why not, but that doesn't look very Rusty 😅 why not `Option<[]>` for the parameter so you can get rid of the constant?

Reply:
Yeh, the constant isn't very Rusty and doesn't really capture any intention. The only reason I used it, is because the method calls are clearer as to why there's a random `&[]` appearing practically everywhere.

I would vote against `Option<&[]>`, because then `None` and `&[]` would mean the same thing, and the implementations will have to `flat_map` them anyway.

Getting rid of the constant makes sense however.